projects/1.18/assets/simple-tomb/simpletomb/lang/zh_cn.json

```
@@ -1 +1,53 @@
-{}
\ No newline at end of file
+{
+    "simpletomb.grave.grave_simple": "简易坟墓",
+    "simpletomb.grave.grave_cross": "十字架坟墓",
+    "simpletomb.grave.grave_normal": "普通坟墓",
+    "simpletomb.grave.tombstone": "墓碑",
+    "item.simpletomb.grave_key": "坟墓钥匙",
+    "simpletomb.commands.null": "子命令格式错误，示例：/simpletomb latest",
+    "simpletomb.commands.latest.null": "未找到最后的坟墓",
+    "simpletomb.message.rip": "R.I.P.",
+    "simpletomb.message.open_grave.success": "坟墓消失了！",
+    "simpletomb.message.open_grave.need_owner": "只有坟墓所有者才能打开它",
+    "simpletomb.message.recover_lost_items": "你已找回丢失的物品",
+    "simpletomb.message.open_grave.need_key": "你需要一把钥匙！",
+    "simpletomb.message.open_grave.wrong_key": "这把钥匙不对！",
+    "simpletomb.message.no_loot_for_grave": "没有墓碑出现，因为死亡时物品栏为空",
+    "simpletomb.message.no_grave_location": "你的物品落了一地；这地方不适合放置坟墓",
+    "simpletomb.message.no_place_for_grave": "没地方放坟墓！",
+    "simpletomb.message.fail_to_place_grave": "坟墓无法放置！",
+    "simpletomb.message.new_grave": "坟墓已创建：%s",
+    "simpletomb.message.journeymap": "[名称：\"坟墓\", x:%d, y:%d, z:%d, 维度：\"%d\"]",
+    "simpletomb.message.no_save_to_restore": "没有需要还原的存档",
+    "simpletomb.message.day": "天数：%d",
+    "simpletomb.message.distance": "距离：%d [%d, %d, %d]，在%d维度中",
+    "_comment": "常规",
+    "simpletomb.config_cat.general": "常规",
+    "simpletomb.config_cat.general.tooltip": "杂项",
+    "simpletomb.config.handle_player_death.tooltip": "模组是否会处理玩家死亡",
+    "simpletomb.config.log_player_grave.tooltip": "是否把玩家的坟墓坐标录入日志",
+    "simpletomb.config.player_grave_access.tooltip": "打开坟墓时是否需要坟墓钥匙",
+    "simpletomb.config.grave_key_on_death.tooltip": "是否在玩家死后给一把坟墓钥匙",
+    "simpletomb.config.decay_time.tooltip": "玩家死亡后多久（分钟）才能不需要坟墓钥匙打开坟墓（-1=禁用）",
+    "simpletomb.config.teleport_dim.tooltip": "允许跨维度传送",
+    "simpletomb.config.pvp_mode.tooltip": "激活PVP模式，使得玩家能打开被他杀死的玩家的坟墓",
+    "simpletomb.config.sniffer_range.tooltip": "生成坟墓时收集物品的半径",
+    "simpletomb.config.ghostly_shape_duration.tooltip": "幽灵形态状态效果的时长（秒）",
+    "simpletomb.config.nerf_ghostly_shape.tooltip": "是否关闭幽灵形态对破坏方块和打开箱子的影响",
+    "simpletomb.config.chance_loss_on_death.tooltip": "一些物品在死亡后直接丢失的概率",
+    "simpletomb.config.percent_loss_on_death.tooltip": "死亡后直接丢失物品的百分比",
+    "simpletomb.config.prevent_death_outside_world.tooltip": "防止在世界边界外死亡",
+    "simpletomb.config.graves_by_pass_griefing_rules.tooltip": "坟墓是否可以绕过anti-griefing规则",
+    "simpletomb.config.unhandled_beneficial_effects": "Unhandled Beneficial Effects",
+    "simpletomb.config.unhandled_beneficial_effects.tooltip": "Beneficial effects that can't used by certain features such as �nkh of pray, lollipop, scroll of preservation, alchemy perk and magic siphon enchantment",
+    "simpletomb.config.unhandled_harmful_effects": "Unhandled Harmful Effects",
+    "simpletomb.config.unhandled_harmful_effects.tooltip": "Harmful effects that can't used by certain features such as tablet of cupidity and the enchantment plague bringer",
```

```suggestion
    "simpletomb.config.unhandled_harmful_effects": "忽略的负面效果",
    "simpletomb.config.unhandled_harmful_effects.tooltip": "无法被某些功能（如贪婪石板和瘟疫使者附魔）所使用的负面效果",
```

https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/5c3b62431da6d96156bf4573d9cf7642f84439b7/projects/1.16/assets/corail-tombstone/tombstone/lang/zh_cn.json#L58

https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/5c3b62431da6d96156bf4573d9cf7642f84439b7/projects/1.16/assets/corail-tombstone/tombstone/lang/zh_cn.json#L563